api/src/main/java/org/pmiops/workbench/api/OfflineUserController.java

```diff
@@ -201,4 +209,26 @@ private boolean timestampsEqual(Timestamp a, Timestamp b) {
     }
     return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
   }
+
+  /**
+   * Audits GCP access for all users in the database.
+   *
+   * This API method is called by a cron job and is not part of our normal user-facing surface.
+   */
+  @Override
+  public ResponseEntity<Void> bulkAuditProjectAccess() {
+    int errorCount = 0;
+    int userCount = 0;
+
+    for (User user : userService.getAllUsers()) {
+      userCount++;
+      List<String> unauthorizedLogs = cloudResourceManagerService.getAllProjectsForUser(user).stream()
+          .filter(project -> !(project.getParent().getId().equals(PMI_OPS_ORG_ID)))
```

Comment:
Have you tried running the cron against our test database?

I'm pretty sure we should see AoU users with roles on projects in firecloud.org, e.g. billing projects created on their behalf. So we may well need to store an environment variable to track the firecloud org ID for each environment, or simpler yet, a constant list of all expected firecloud org IDs.

In fact, for my fake-research-aou user, I don't see any project-level roles on projects in PMI-Ops.org when I run the projects.list command.

Reply:
https://stackoverflow.com/questions/4913891/how-to-run-gae-cron-jobs-as-specific-app-version

It looks like we can't run crons without merging it in against a versioned deployment. I think the only way to actually test out the cron is to merge it in and make sure it runs then.

Reply:
I have no clue what orgId we should be seeing/whitelisting, but I figure for now, we might log extra. The orgId I whitelisted here is what I got when I ran the query authed as my test user, and is what showed up for this sort of project: 
{
      "projectNumber": "731830181574",
      "projectId": "aou-rw-test-614",
      "lifecycleState": "ACTIVE",
      "name": "aou-rw-test-614",
      "labels": {
        "billingaccount": "fc-pmi-drc-workbench-testing"
      },
      "createTime": "2019-06-06T20:05:13.162Z",
      "parent": {
        "type": "organization",
        "id": "400176686919"
      }
    },

Reply:
It's definitely possible to trigger a cron execution from a local machine -- see https://github.com/all-of-us/workbench/blob/master/api/doc/cron-jobs.md#local-testing, I found this extremely helpful when iterating on some of the other crons.

Reply:
Ah – it may have been a simple naming issue, then. The aou-rw-test-614 project is part of the FireCloud org, since that's where user projects live. So s/PMI_OPS/FIRECLOUD/ might be enough to make me happy :)

Reply:
Oh, I totally tested the endpoint. Sorry, I thought this meant testing the cron execution itself, as triggered by appengine. 

Yes, I have run it locally, and curled the endpoint.